include/netutils/md5.h

```diff
@@ -10,7 +10,6 @@
  *
  *   This code implements the MD5 message-digest algorithm.
  *   The algorithm is due to Ron Rivest.  This code was
- *   written by Colin Plumb in 1993, no copyright is claimed.
```

Comment:
Did you intend to remove this line?

Reply:
No, I didn't want to remove it and also the above line will become incomplete if it was removed.